pkg/kubelet/container/runtime.go

```diff
@@ -433,8 +431,6 @@ type RunContainerOptions struct {
 	Mounts []Mount
 	// The host devices mapped into the containers.
 	Devices []DeviceInfo
-	// The port mappings for the containers.
-	PortMappings []PortMapping
```

Comment:
simplifying code review: port mapping is needed per entire pod. Not needed per-container.